Source/ACE.Entity/Vector3.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.IO;
+using System.Numerics;
+
+namespace ACE.Entity
+{
+    public class AceVector3
```

Comment:
3 private floats, that you can't get out once set?  What's the point?  What does this object give us that the existing Vector3 object doesn't?